### ISSUE TITLE
HtModal

### ISSUE THREAD

**Issue by dpgraham4401** [USEPA/haztrak#204]
# 🚀 Feature Request

Many of the components in ManifestForm use a modal to bring up a sub-form. It would be great to eliminate some of the boilerplate and refactor these into a new component `HtModal` which will just wrap around react-bootstrap's Modal components and apply some default styling behavior. 

as an example, see HtCard, which does a similar things with react-bootstrap's card component. It should pass react-bootstraps ModalProps (i think that's what it's called, would need to double check). Any props passed should override the Haztrak's default style/behavior.

**Comment by Badrri-Narayanan** [comment]
Hi @dpgraham4401 , can you please assign me this task?

**Comment by dpgraham4401** [comment]
let me know if you need anything

**Comment by Badrri-Narayanan** [comment]
sure thanks

**Comment by Badrri-Narayanan** [comment]
@dpgraham4401 can please let me know how I can access the existing ManifestForm view. The following screenshot contains the dashboard view when I am logged in as **testuser1**. I just want to understand the current functionality.
![image](https://user-images.githubusercontent.com/26011119/210586960-2cd88241-cf20-4c51-93c7-123b181cd97e.png)

#### My Site page view for testuser1
There too the list of sites is empty. Could it be that it gets called from here?

![image](https://user-images.githubusercontent.com/26011119/210587413-6bd131ca-f50c-45bc-849f-2bf51bdcb489.png)

I will be grateful for your help and guidance!

**Comment by dpgraham4401** [comment]
@Badrri-Narayanan 
TLDR as a workaround, navigate to 
http://localhost:3000/manifest/draft/edit

I need to address #281 

Since, in the current phase, we don't have a deployed version of haztrak, there's only two ways to get data in there. 

1.  Add a fixture (like testuser1) that's added on startup
2. Pull the data from RCRAInfo, the EPA web application this project is building against.
3. manually supply it via http://localhost:8000/admin

Since there are no dummy data in there, this is leaving you with nothing to click on to go to.

We had some, but recently removed it because I anticipated having better quality starter data in there soon.
